sdcm/cluster.py

```diff
@@ -4307,6 +4307,16 @@ def _wait_for_nodes_up_and_normal():
 
         _wait_for_nodes_up_and_normal()
 
+    @retrying(n=360, sleep_time=10, allowed_exceptions=ClusterNodesNotReady, message="Waiting for CQLPing to return")
+    def wait_native_transport(self, node):
```

Comment:
Not sure I know what this API means...

But why not implement it on BaseNode ?

Reply:
in one hand, this is part of the node service itself, but in other hand, this is related to how that node is serving the cluster...
if you think this is necessary, i can move it to the BaseNode

Reply:
about "not knowing what this API means", we could use instead some `node.run_cqlsh(<some query>)` instead

Reply:
@fruch moved to the `BaseNode`